cli/pkg/kctrl/cmd/package/repository/add_or_update.go

```diff
@@ -141,6 +145,30 @@ func (o *AddOrUpdateOptions) Run(args []string) error {
 		return err
 	}
 
+	if o.CreateNamespace {
+		o.statusUI.PrintMessagef("Creating namespace '%s'", o.NamespaceFlags.Name)
+
+		coreClient, err := o.depsFactory.CoreClient()
+		if err != nil {
+			return err
+		}
+
+		namespace := &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: o.NamespaceFlags.Name,
+			},
+		}
+
+		_, err = coreClient.CoreV1().Namespaces().Create(context.Background(), namespace, metav1.CreateOptions{})
+		if err != nil {
+			if errors.IsAlreadyExists(err) {
+				o.statusUI.PrintMessagef("The namespace '%s' already exists", o.NamespaceFlags.Name)
+			} else {
+				return err
+			}
```

Comment:
```suggestion
			if !errors.IsAlreadyExists(err) {
			         return err
			} 
			o.statusUI.PrintMessagef("The namespace '%s' already exists", o.NamespaceFlags.Name)
```

Reply:
I have refactored that part to print only a message if the namespace is created as suggested in the previous comment.